lib/src/cli/atlas_apps/deployapps_command.dart

```diff
@@ -67,9 +67,13 @@ class DeployAppsCommand extends Command<void> {
     final apps = await client.getOrCreateApps();
 
     print('App import is complete. There are: ${apps.length} apps on the server:');
+    List<String> listApps = [];
     apps.forEach((_, value) {
       print("  App '${value.name}': '${value.clientAppId}'");
+      listApps.add(value.appId);
     });
+    print("appIds: ");
+    stdout.write(listApps.join(","));
```

Comment:
Why is this using stdout instead of print?

Reply:
I didn't know I would be able to read the output from print. It is the same actually. I can change it.